style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: rgb(141, 107, 56);
+}
+
+.classbody {
+  background-color: rgb(211, 162, 82);
+  width: 432px;
+  margin-left: 500px;
+}
+
+h2 {
+  font-family: "Anton", sans-serif;
+  margin-left: 50px;
+}
+
+p {
+  font-family: "Noto Sans JP", sans-serif;
+  text-transform: uppercase;
+  margin-left: 50px;
+}
+
+.small-logo {
+  float: right;
+  margin-right: 20px;
+}
+
+a {
+  font-family: "Noto Sans JP", sans-serif;
+  text-transform: uppercase;
+}
+
+.hashtag,
+.buttons {
+  text-decoration: none;
+}
```

Comment:
You're using two different selectors (the `a` element on line 27 and the class names on lines 32 & 33), but they all affect exactly the same things. Consider combining these rules under one type of selector.